database/migrations/2018_07_12_145513_create_managers_table.php

```diff
@@ -15,6 +15,7 @@ public function up()
 		Schema::create('managers', function(Blueprint $table)
 		{
 			$table->increments('id');
+			$table->string('government_email')->nullable();
```

Comment:
I don't think we can edit a previous migration with a new column, it won't get run in production